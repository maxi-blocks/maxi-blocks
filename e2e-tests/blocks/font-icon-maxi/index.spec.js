/**
 * WordPress
 */
import {
	createNewPost,
	insertBlock,
	getEditedPostContent,
} from '@wordpress/e2e-test-utils';

describe('Font Icon Maxi', () => {
	beforeEach(async () => {
		await createNewPost();
	});

	it('Font Icon Maxi does not break', async () => {
		await insertBlock('Font Icon Maxi');

		expect(await getEditedPostContent()).toMatchSnapshot();
	});
});
