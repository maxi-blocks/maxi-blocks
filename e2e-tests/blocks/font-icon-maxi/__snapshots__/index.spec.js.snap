// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Font Icon Maxi Font Icon Maxi does not break 1`] = `
"<!-- wp:maxi-blocks/font-icon-maxi {\\"blockStyle\\":\\"maxi-light\\",\\"uniqueID\\":\\"font-icon-maxi-12\\",\\"isFirstOnHierarchy\\":true} -->
<div class=\\"wp-block-maxi-blocks-font-icon-maxi maxi-motion-effect maxi-block maxi-font-icon-block maxi-light font-icon-maxi-12\\" id=\\"font-icon-maxi-12\\"><div class=\\"maxi-background-displayer\\"></div></div>
<!-- /wp:maxi-blocks/font-icon-maxi -->"
`;
